bake/remote.go

```diff
@@ -23,13 +25,42 @@ type Input struct {
 }
 
 func ReadRemoteFiles(ctx context.Context, nodes []builder.Node, url string, names []string, pw progress.Writer) ([]File, *Input, error) {
-	var session []session.Attachable
+	var sessions []session.Attachable
 	var filename string
+
 	st, ok := dockerui.DetectGitContext(url, false)
 	if ok {
-		ssh, err := controllerapi.CreateSSH([]*controllerapi.SSH{{ID: "default"}})
-		if err == nil {
-			session = append(session, ssh)
+		var sshSpecs []*controllerapi.SSH
+		if v := os.Getenv("BUILDX_BAKE_GIT_SSH"); v != "" {
```

Comment:
ID of this needs to be set with `MountSSHSock()` if not default. Setting it to other ID would not work otherwise. So probably it should just be the path.